### PR TITLE
FEAT-#54 Add unidist main logo image

### LINKED ISSUE #54: Logo design
Update the repo with new logos

### PR DESCRIPTION
**Pull request by no-ponomarev**
## What do these changes do?

Add logo to the README

- [ ] passes `flake8 .`
- [ ] passes `black .`
- [ ] signed commit with `git commit -s` 
- [x] Resolves #54   
- [ ] tests added and passing
- [ ] module layout described at `docs/developer/architecture.rst` is up-to-date

### CODE REVIEW

**Comment by prutskov** [comment]
Additionally, logo should be added to main page of rst docs. Do we have only `.png`? `.svg` format would be more flexible.

**Comment by YarShev** [comment]
@no-ponomarev, please fix commit message.

File: README.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+<p align="center">
+    <a href="https://unidist.readthedocs.io"><img alt="" src="docs/img/unidist-logo-simple-628x128.png?raw=true"></a>
+</p>
+
```
Comment (YarShev) [inline README.md:4]:
```suggestion
```

**Comment by YarShev** [comment]
I do not think we are going to change the logo so png is enough.

**Comment by prutskov** [comment]
> I do not think we are going to change the logo so png is enough.

The reason by what I propose to use `.svg` is not future changes of the logo design. `png` format has fixed resolution, but `svg` adapts to every resolutions.

**Comment by no-ponomarev** [comment]
> > I do not think we are going to change the logo so png is enough.
> 
> The reason by what I propose to use `.svg` is not future changes of the logo design. `png` format has fixed resolution, but `svg` adapts to every resolutions.

Figma can export to `svg` also. As I see, modin logo is stored in `png` too. It is a default transparent format for images.

**Comment by no-ponomarev** [comment]
> Additionally, logo should be added to main page of rst docs. Do we have only `.png`? `.svg` format would be more flexible.

Added on the getting started page

File: docs/index.rst [hunk 2]
```diff
@@ -3,6 +3,10 @@
 
       SPDX-License-Identifier: Apache-2.0
 
+.. image:: img/unidist-logo-simple-628x128.png
```
Comment (YarShev) [inline docs/index.rst:6]: Let's also set `html_favicon` and `html_logo` in conf.py.

https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-html_logo
https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-html_favicon

**Review by YarShev — APPROVED** [review]
Looks great! LGTM!

**Review by prutskov — APPROVED** [review]

**Pull request merged**